src/Surfnet/AzureMfa/Application/Service/AzureMfaService.php

```diff
@@ -38,6 +38,8 @@
  */
 class AzureMfaService
 {
+    const SAML_EMAIL_ATTRIBUTE = 'urn:mace:dir:attribute-def:emailAddress';
```

Comment:
That does not exist in mace-dir. The emailadresss is in `...:mail`. See: https://www.internet2.edu/products-services/trust-identity/mace-registries/urnmace-namespace/urn-mace-dir-registry/urn-mace-dir-attribute-def/